README.md

```diff
@@ -1,30 +1,39 @@
-<h1 align="center">OpenST - Empowering Decentralized Economies</h1>
+<h1 align="center">Branded Tokens - Tokenizing Mainstream Applications</h1>
 
-[![Gitter: JOIN CHAT](https://img.shields.io/badge/gitter-JOIN%20CHAT-brightgreen.svg)](https://gitter.im/OpenSTFoundation/SimpleToken)
+[![Discourse: JOIN DISCUSSION](https://img.shields.io/discourse/https/discuss.openst.org/topics.svg?style=flat)](https://discuss.openst.org/) [![Travis CI: DEVELOP](https://img.shields.io/travis/OpenSTFoundation/brandedtoken-contracts/develop.svg?style=flat)](https://travis-ci.org/OpenSTFoundation/brandedtoken-contracts)
 
-OpenST blockchain infrastructure empowers new economies for mainstream businesses and emerging (D)Apps. The token and other smart contracts in this repository are intended for use with the OpenST Protocol, a framework for tokenizing businesses.
+A Branded Token allows a mainstream application to create a value-backed token designed specifically for its application's context. A Branded Token implements the required and optional [EIP-20 Standard Token interface](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md).
 
-_While this software is available as-is for anyone to use, we caution that this is in early stage and under heavy ongoing development and improvement. Please report bugs and suggested improvements._
+Holders of branded tokens must be able to redeem the value that backs those branded tokens. Branded tokens are only usable within the application context. The application can maintain a policy on accepting new holders.
 
-## OpenST Tokens
+A Utility Branded Token, which also implements the EIP-20 Standard Token interface, is the utility token representation of the Branded Token on a sidechain. This representation is orchestrated via a gateway, comprising a Gateway contract on the same chain as the Branded Token, and a CoGateway contract on the sidechain with the Utility Branded Token.
 
-Utility Branded Tokens and Value Branded Tokens form the basis of token economies built with the OpenST Protocol.
+A composer is a contract that can be used to optimize the transactions required to perform an action. A Gateway Composer, a type of composer, facilitate's staking value for branded tokens and minting a utility representation of those branded tokens with a Utility Branded Token through a gateway for use within the given application, thereby reducing both the number of transactions to execute and the number of contracts to call in order to stake and mint.
```

Comment:
If we can rename composer => "Gateway Composer" at the beginning of introduction . We are following everywhere as a practice.
```
A Gateway Composer is a contract that can be used to optimize
```
Since it's the beginning of GC statement, it would be appropriate to use "Gateway Composer" here.

Reply:
I agree that it would be appropriate and consistent with this readme. However, I think strictly following that pattern in this case risks more than it benefits.

I am happy to add the requested language, because it adds nice color, but it is really about the UX Composer concept. So, I chose this approach to satisfy your request to bring in that color but without unnecessarily expanding any formal descriptions (and risking the readme going out of date if the UX Composer description changes) or stealing thunder from the OIPs repository, which is where the UX Composer concept should first be introduced.

However, if you think this approach appears inconsistent and confusing, I can remove the added optimization language. Or I could move the optimization language to the end of that paragraph, but I fear that undermines the value the language adds.

Reply:
Fair enough. We can keep as it is.